package/boot/arm-trusted-firmware-rockchip/Makefile

```diff
@@ -67,6 +70,22 @@ ifeq ($(BUILD_VARIANT),rk3399)
     M0_CROSS_COMPILE=$(PKG_BUILD_DIR)/$(M0_GCC_NAME)-$(M0_GCC_RELEASE)-$(M0_GCC_VERSION)/bin/arm-none-eabi-
 endif
 
+ifeq ($(BUILD_VARIANT),rk3588)
+  TPL_FILE:=rk3588_ddr_lp4_2112MHz_lp5_2400MHz_v1.18.bin
```

Comment:
Maybe rename rk3588 to rk3588-tpl in rkbin so we don't need to download this separately?

Reply:
Great idea!